src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -79,6 +79,11 @@ public enum NuGetLogCode
         /// </summary>
         NU1007 = 1007,
 
+        /// <summary>
+        /// Project style specified is not a valid value.
+        /// </summary>
+        NU1008 = 1008,
```

Comment:
We'll need new documentation for this [here](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings).

Reply:
There are a bunch of missing codes in the documentation:
``` C#
/// <summary>
/// Locked mode, but restore needs to update the lock file.
/// </summary>
NU1004 = 1004,

/// <summary>
/// Invalid combination of RestorePackagesWithLockFile and packages.lock.json file.
/// </summary>
NU1005 = 1005,

/// <summary>
/// NuGet configuration file has an invalid package source value.
/// </summary>
NU1006 = 1006,

/// <summary>
/// Project provided runtime graph is invalid. Either does not exist or cannot be parsed.
/// </summary>
NU1007 = 1007,

/// <summary>
/// Project style specified is not a valid value.
/// </summary>
NU1008 = 1008,
```

There should probably be an effort to get them all up-to-date